src/networking.c

```diff
@@ -4292,6 +4292,34 @@ int stopThreadedIOIfNeeded(void) {
     }
 }
 
+/* This function increases the number of threads to server.io_threads_num.
+ * It returns C_OK when the update is successful, otherwise it returns
+ * C_ERR when trying to decrease the number of threads. */
+int updateThreadedIO(void) {
```

Comment:
Maybe we change the name to `upgrade`?
```suggestion
int upgradeThreadedIO(void) {
```

Reply:
updateThreadedIO goes better with the other functions that change configurations as well. Also, the function can later be adjusted to reduce the number of threads if needed, but it's up to the members to decide which one they prefer.

Reply:
i think the function name should be kept, but the code should be updated to also support reducing the number of threads, not just increasing it.